mscore/playpanel.cpp

```diff
@@ -333,6 +339,19 @@ void PlayPanel::tempoSliderPressed(int)
       {
       tempoSliderIsPressed = true;
       }
+//---------------------------------------------------------
+//   setVolume
+//---------------------------------------------------------
+      
+void PlayPanel::volLabel()
+      {
+      qDebug()<<"Volume value ="<<vol;
+      if (vol < 0.001)
+            vol=qreal(-200);
```

Comment:
Another magic number? This one leads to a bug when you open play panel first time:
<img width="1515" alt="Untitled 2019-03-18 15-09-46" src="https://user-images.githubusercontent.com/18065034/54532569-b1494c80-4990-11e9-952c-ee14fbbf48b7.png">